indexer-opts/sqlx-data.json

```diff
@@ -0,0 +1,23 @@
+{
+  "db": "PostgreSQL",
+  "28a0bf659e47bca17cf8a9d951489819d2dda115cd3f83b50019cb811d5ead5a": {
+    "describe": {
+      "columns": [
+        {
+          "name": "last_processed_block_height",
+          "ordinal": 0,
+          "type_info": "Numeric"
+        }
+      ],
+      "nullable": [
+        false
+      ],
+      "parameters": {
+        "Left": [
+          "Text"
+        ]
+      }
+    },
+    "query": "\nSELECT last_processed_block_height FROM meta WHERE indexer_id = $1 LIMIT 1\n                    "
```

Comment:
Do we need this?

Reply:
This is required for offline `cargo check`, it's like mock. When we introduce the CI with a check it'll be required.

Reply:
Can you share more info somewhere, how to generate such files, when do we need to run `cargo check`, etc